### DISCUSSION TITLE
Tracking, monitoring of user actions.

### DISCUSSION THREAD

**Discussion by jekeam** [python-telegram-bot/python-telegram-bot#3174] [Q&A] [answered]
How can I track all interactions with the bot? I want to record the time of the last user activity, how can I do this without changing the code of the written functions.

**Comment by Poolitzer** [comment, marked as answer]
An update type handler in a different group will do the trick.

**Reply by jekeam** [reply]
Maybe there is an example? I am new to this. Thank You!

**Reply by jekeam** [reply]
Thanks, this example helped me understand: https://github.com/python-telegram-bot/python-telegram-bot/discussions/2380
